### ISSUE TITLE
Add SC3044 to documentation

### ISSUE THREAD

**Issue by hackallcode** [hadolint/hadolint#765]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

I use binary for Linux from release v2.8.0.
Linter shows `SC3044 warning: In POSIX sh, 'popd' is undefined.` warning in some cases,
but there is no such check in the documentation of the master branch or of the 2.8.0 tag.

### Actual behavior

All warnings from the release are in the documentation.

### Steps to reproduce the behavior

- [download](https://github.com/hadolint/hadolint/releases/download/v2.8.0/hadolint-Linux-x86_64) binary for Linux from release 2.8.0
- lint Dockerfile with `popd` usage
- you will get a warning like this:

  ```bash
  Dockerfile.build:217 SC3044 warning: In POSIX sh, 'popd' is undefined.
  ```

**Output of `hadolint --version`:**

```bash
Haskell Dockerfile Linter UNKNOWN
```

**Dockerfile**

```Dockerfile
FROM centos:7 as centos
...
RUN popd
```

**Additional information**

Linter was run from CentOS Linux 7 in Docker.

**Comment by m-ildefons** [comment]
Hi @hackallcode ,
unlike the warnings with a `DLxxxx` code, all errors, warnings, etc. with an `SCxxxx` code emitted by [Shellcheck](https://github.com/koalaman/shellcheck) and further documentation can be found in its [wiki](https://github.com/koalaman/shellcheck/wiki/SC3044).

Since changes to the behavior of those checks and their documentation respectively is done through the Shellcheck project, there is no need for additional documentation in this project. I'm sure you can also see that creating and maintaining such documentation as part of this project would add a lot of work with little benefit.
